config/locales/forms/registration_received_pending_worldpay_payment_forms/en.yml

```diff
@@ -0,0 +1,11 @@
+en:
+  waste_carriers_engine:
+    registration_received_pending_worldpay_payment_forms:
+      new:
+        title: "We’re processing your payment"
+        heading: "We’re processing your payment"
+        highlight_text: "Please use this registration number if you contact us:"
+        paragraph_1: "We've sent a confirmation email to %{email}."
+        subheading_1: "We have received your application"
+        paragraph_2: "We’re currently processing your payment and will let you know when it has been received."
+        paragraph_3: "Your registration will not be active until we have confirmed your registration"
```

Comment:
```suggestion
        paragraph_3: "Your registration will not be active until we have confirmed your registration."
```

Reply:
there is no dot in the wireframe for this

Reply:
Sorry, didn't realise this was the bit of text in the panel! I thought it was just a regular paragraph. Ignore me!